src/key/dynamic_nostd.rs

```diff
@@ -1,28 +1,26 @@
-
-use alloc::String;
 use alloc::borrow::Cow;
-use alloc::Clone;
+use alloc::string::String;
 use alloc::string::ToString;
+use core::clone::Clone;
 
-use core::convert::{From,Into,AsRef};
-use core::ops::Deref;
-use core::str::FromStr;
 use core::cmp::PartialEq;
-use core::hash::{Hash,Hasher};
-use core::iter::{FromIterator, IntoIterator};
+use core::convert::{AsRef, From, Into};
 use core::fmt;
+use core::hash::{Hash, Hasher};
+use core::iter::{FromIterator, IntoIterator};
+use core::ops::Deref;
+use core::str::FromStr;
```

Comment:
Fixing import ordering feels like an unrelated change. We should probably implement it in a separate change.